examples/tutorials/self-paced-training/part-4_advanced_federated_learning/chapter-10_federated_XGBoost/10.0_introduction/introduction.ipynb

```diff
@@ -1,19 +1,71 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "cae3fb2d-8949-4e6d-a2af-0282ee973285",
+   "metadata": {},
+   "source": [
+    "# Federated Learning for XGBoost \n",
+    "This chapter demonstrates how to use NVFlare to train an XGBoost model in a federated learning setting. \n",
+    "Several potential variations of federated XGBoost are illustrated, including:\n",
+    "- non-secure horizontal collaboration with histogram-based and tree-based mechanisms.\n",
+    "- non-secure vertical collaboration with histogram-based mechanism.\n",
+    "- secure horizontal and vertical collaboration with histogram-based mechanism and homomorphic encryption.\n",
+    "\n",
+    "Let's first visit the basics of XGBoost and the collaboration modes."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6da8f899-2804-4d6e-ab03-a02ce115d32f",
+   "metadata": {},
+   "source": [
+    "## XGBoost \n",
+    "XGBoost is a machine learning algorithm that uses decision/regression trees to perform classification and regression tasks, \n",
+    "mapping a vector of feature values to its label prediction. It is especially powerful for tabular data, so even in the age of LLM, \n",
+    "it still widely used for many tabular data use cases. It is also preferred for its explainability and efficiency.\n",
+    "\n",
+    "In these examples, we use [DMLC XGBoost](https://github.com/dmlc/xgboost), which is an optimized distributed gradient boosting library. \n",
+    "It offers advanced features like GPU accelerated capabilities, and distributed/federated learning support.\n",
+    "\n",
+    "## Collaboration Modes and Data Split\n",
+    "Essentially there are two collaboration modes: horizontal and vertical:\n",
+    "- In horizontal case, each participant has access to the same features (columns) of different data samples (rows). \n",
+    "In this case, everyone holds equal status as \"label owner\"\n",
+    "- In vertical case, each client has access to different features (columns) of the same data samples (rows).\n",
+    "We assume that only one is the \"label owner\" (or we call it as the \"active party\")\n",
+    "\n",
+    "To simulate the above two collaboration modes, we split the dataset both horizontally and vertically, and \n",
+    "we give site-1 the label column for simplicity.\n",
+    "\n",
+    "## Federated Training of XGBoost\n",
+    "Continue with this chapter for two scenarios:\n",
+    "### [Federated XGBoost without Encryption](../10.1_fed_xgboost/fed_xgboost.ipynb)\n",
+    "This section includes instructions on running federated XGBoost without encryption under histogram-based and tree-based horizontal \n",
```

Comment:
suggest:

"This section provides instructions for running federated XGBoost without **homomorphic encryption,** covering both histogram-based and tree-based horizontal collaboration, as well as histogram-based vertical collaboration."

Reply:
updated